src/net/sourceforge/kolmafia/session/TrainsetManager.java

```diff
@@ -0,0 +1,198 @@
+package net.sourceforge.kolmafia.session;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import net.sourceforge.kolmafia.preferences.Preferences;
+
+public class TrainsetManager {
```

Comment:
You can add a private constructor which will stop this class being considered uncovered